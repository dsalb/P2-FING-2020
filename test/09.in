# Implementar longitudesCaminosMasCortos
longitudesCaminosMasCortos 5
hacerVecinos 2 5 30
longitudesCaminosMasCortos 2
longitudesCaminosMasCortos 5
hacerVecinos 8 5 20
longitudesCaminosMasCortos 2
longitudesCaminosMasCortos 5
longitudesCaminosMasCortos 8
hacerVecinos 1 4 10
hacerVecinos 5 4 10
longitudesCaminosMasCortos 1
longitudesCaminosMasCortos 2
longitudesCaminosMasCortos 4
longitudesCaminosMasCortos 5
longitudesCaminosMasCortos 8
hacerVecinos 10 4 90
longitudesCaminosMasCortos 1
longitudesCaminosMasCortos 2
longitudesCaminosMasCortos 3
longitudesCaminosMasCortos 4
longitudesCaminosMasCortos 5
longitudesCaminosMasCortos 6
longitudesCaminosMasCortos 7
longitudesCaminosMasCortos 8
longitudesCaminosMasCortos 9
longitudesCaminosMasCortos 10
Fin
