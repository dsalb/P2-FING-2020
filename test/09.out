1>#  Implementar longitudesCaminosMasCortos.
2>INF INF INF INF 0.00 INF INF INF INF INF 
3>Nuevos vecinos: 2 5 30.00.
4>INF 0.00 INF INF 30.00 INF INF INF INF INF 
5>INF 30.00 INF INF 0.00 INF INF INF INF INF 
6>Nuevos vecinos: 8 5 20.00.
7>INF 0.00 INF INF 30.00 INF INF 50.00 INF INF 
8>INF 30.00 INF INF 0.00 INF INF 20.00 INF INF 
9>INF 50.00 INF INF 20.00 INF INF 0.00 INF INF 
10>Nuevos vecinos: 1 4 10.00.
11>Nuevos vecinos: 5 4 10.00.
12>0.00 50.00 INF 10.00 20.00 INF INF 40.00 INF INF 
13>50.00 0.00 INF 40.00 30.00 INF INF 50.00 INF INF 
14>10.00 40.00 INF 0.00 10.00 INF INF 30.00 INF INF 
15>20.00 30.00 INF 10.00 0.00 INF INF 20.00 INF INF 
16>40.00 50.00 INF 30.00 20.00 INF INF 0.00 INF INF 
17>Nuevos vecinos: 10 4 90.00.
18>0.00 50.00 INF 10.00 20.00 INF INF 40.00 INF 100.00 
19>50.00 0.00 INF 40.00 30.00 INF INF 50.00 INF 130.00 
20>INF INF 0.00 INF INF INF INF INF INF INF 
21>10.00 40.00 INF 0.00 10.00 INF INF 30.00 INF 90.00 
22>20.00 30.00 INF 10.00 0.00 INF INF 20.00 INF 100.00 
23>INF INF INF INF INF 0.00 INF INF INF INF 
24>INF INF INF INF INF INF 0.00 INF INF INF 
25>40.00 50.00 INF 30.00 20.00 INF INF 0.00 INF 120.00 
26>INF INF INF INF INF INF INF INF 0.00 INF 
27>100.00 130.00 INF 90.00 100.00 INF INF 120.00 INF 0.00 
28>Fin.
